starwars/src/components/CharacterList.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import Character from "./Character";
+
+function CharacterList(props) {
+  return props.characterArray.map(character => (
+    <Character
```

Comment:
You see, how there's no `<div>` wrapper in here? That's what I meant with my previous comment 😅